src/StoryDeck.elm

```diff
@@ -58,37 +58,18 @@ storyTeaser deckId =
         t =
             toString
     in
-    div [ class "card" ]
-        [ a
-            [ href ("#/stories/" ++ String.fromInt deckId)
-            , onClick (ButtonPress "story" "view-single" (t (storyTitle deckId)) False)
-            ]
-            [ img [ class "card--thumbnail", src (storyTeaserImgPath deckId), alt (t (storyTeaserImgAltText deckId)) ] []
-            ]
-        , a
-            [ class "link--unstyled"
-            , href ("#/stories/" ++ String.fromInt deckId)
-            , onClick (ButtonPress "story" "view-single" (t (storyTitle deckId)) False)
-            ]
-            [ h3 [ class "title--small" ] [ text (t (storyTitle deckId)) ]
-            ]
+    a
```

Comment:
`div` element with `onClick` to cause programmatic navigation change here.

Reply:
Semantically this should be a link (maybe?) for accessibility. I know you can trigger an onClick on a div, but we probably want this to be what it says on the tin if possible. Is there an aria role you'd add if we need to keep it as a div?

@falconx Will know more than I whether is ok to wrap an anchor around these elements - particularly the h3 I am thinking of, but I think one of the reasons it is allowed in html5 is to make more semantic mark-up for screenreaders.

Happy to approve either way - but the 2 of you should have a conversation over reasons why one way and not the other. It's not my area.